scripts/stability-report.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# generates a csv that includes a report of module stability and cfn-only status
+# usage: repo-root$ scripts/stability-report.sh
+set -euo pipefail
+outfile="stability.csv"
```

Comment:
IMO, this would be more usable if it output to `STDOUT` instead of to some hard-coded file.

Reply:
Done.